compiler/src/iree/compiler/Dialect/Flow/IR/FlowOps.td

```diff
@@ -964,7 +964,9 @@ def FLOW_CallOp : FLOW_Op<"call", [
     Variadic<AnyType>:$arguments,
     FLOW_ShapeDynamicDims:$argument_dims,
     FLOW_ShapeDynamicDims:$result_dims,
-    OptionalAttr<Util_TiedOpStorageAttr>:$tied_operands
+    OptionalAttr<Util_TiedOpStorageAttr>:$tied_operands,
+    OptionalAttr<DictArrayAttr>:$arg_attrs,
+    OptionalAttr<DictArrayAttr>:$res_attrs
```

Comment:
I do not update the printer and parser in the PR. It makes me think that if we should just pass `nullptr` to builders everywhere. We only update the code when there is a need.

The status of the upstream change does not add any IR roundtrip test, but it has few attribute propagation changes.

Reply:
I think that's ok for now - we'll get to it at some point (arg/result attrs would be useful on ops and may be a way on functions to denote expected encodings and stuff)